docs/Basic-Guide.md

```diff
@@ -63,15 +78,25 @@ More information and documentation is provided in the
 
 ### Adding a Brain to the training session
 
-Since we are going to build this environment to conduct training, we need to add 
-the Brain to the training session. This allows the Agents linked to that Brain
-to communicate with the external training process when making their decisions.
-
-1. Assign the **3DBallLearning** to the agents you would like to train and the **3DBallPlayer** Brain to the agents you want to control manually. 
-   __Note:__ You can only perform training with an `Learning Brain`.
+To set up the environment for training, you will need specify which agents are contributing
+to the training and which Brain is being trained. You can only perform training with
+a `Learning Brain`.
+
+1. Assign the **3DBallLearning** Brain to the agents you would like to train.  
+   __Note:__ You can assign the same brain to multiple agents at once : To do so, you can
+   use the prefab system. When an agent is created from a prefab, modifying the prefab 
+   will modify the agent as well. If the agent does not synchronize with the prefab, you
+   can hit the Revert button on top of the Inspector.
+   Alternatively, you can select multiple agents in the scene and modify their `Brain`
+   property all at once.
 2. Select the **Ball3DAcademy** GameObject and make sure the **3DBallLearning** Brain
    is in the Broadcast Hub. In order to train, you need to toggle the
    `Control` checkbox.
+   
+__Note:__ Assigning a Brain to an agent (dragging a Brain into the `Brain` property of 
+the agent) means that the Brain will be making decision for that agent. Whereas dragging
+a Brain into the Broadcast Hub means that the Brain will be exposed to the Python process.
```

Comment:
Maybe change to:

Whereas dragging a Brain into the Broadcast Hub means that the Brain will be exposed to the Python process, and checking the Control toggle means the Brain will be exposed to the Python process and controlled by the Python process(so that the training happens).